transaction-requests-service/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for central-settlement.
+# This is a YAML-formatted file.
+
+# Declare global configurations
+global:
+  config:
+    ## Pod scheduling preferences.
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ##
+    affinity: {}
+
+    ## Node labels for pod assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+    nodeSelector: {}
+
+    ## Set toleration for schedular
+    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    tolerations: []
+
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: mojaloop/transaction-requests-service
+  tag: v8.3.0-snapshot
```

Comment:
This docker image does not seem to be published to the mojaloop docker repo?

Please check the license check. Also did you verify that this deploys correctly when testing the deployment?

Reply:
fixed